sdk/storage/azure-storage-file-share/tests/perfstress_tests/download.py

```diff
@@ -0,0 +1,35 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure_devtools.perfstress_tests import get_random_bytes
+
+from ._test_base import _ShareTest
+
+
+class DownloadTest(_ShareTest):
```

Comment:
Same feedback as blob tests, download should re-use server file across all instances, while upload each instance should use its own server file.

Reply:
They are all downloading the same file? Only one upload is done in `global_setup`? Or am I misunderstanding?

Reply:
You are correct, I had it backwards.